### ISSUE TITLE
Enable approval of foreign tickets in collect mode

### ISSUE THREAD

**Issue by kyreyt** [microsoft/vsts-extension-retrospectives#343]
On Retrospective board of Azure devOps in Collect mode allow approval of foreign tickets if you would like to mention the same idea.
As possible solution "finger-up" or "+" sign can be added to each created ticket.  

Use case: during collection of ideas in Retrospective different team-members can have the same ideas or problems. In order to avoid issues-duplication they can just press "finger-up" or "+" sign on this ticket to demonstrate that it is importent for few members in the team.

**Comment by mindlessroman** [comment]
@Tatakyr could you add a bit more context? Is there more needed than grouping the similar cards and then voting on the group?

**Comment by Tatakyr** [comment]
> @Tatakyr could you add a bit more context? Is there more needed than grouping the similar cards and then voting on the group?

@mindlessroman, not really. Usually voting is mostly concentrated on problems which should be solved. However very often team have similar positive issues, but do not write  them few times and do not mention it few times during discussion. However it is very significant to know that one or another "ticket" do matter for numerous team-members.
